src/pydrex/minerals.py

```diff
@@ -336,19 +336,14 @@ def apply_gbs(orientations, fractions, config):
                 )
             ),
             time_end,
-            # first_step=max_step / 4,  # TODO: Move divisor to config?
-            max_step=max_step,
-            # atol=np.inf,  # FIXME: Different solver or smart way to set tolerance.
+            # first_step=max_step / 4,
+            # max_step=max_step,
+            rtol=1e-6,
```

Comment:
You may want to set `atol` to 0. The reasoning here is that `atol` can bypass `rtol` if the values evaluated in the integration process are smaller than `atol`.

Reply:
Have set `atol` to 1e-99, I found that 0 caused occasional crashes depending on n_grains/timesteps.